types/react-split-pane/index.d.ts

```diff
@@ -26,6 +26,9 @@ declare namespace ReactSplitPane {
         onDragFinished?: Function;
         onDragStarted?: Function;
         primary?: string;
+        onResizerClick?: Function;
```

Comment:
We should try and narrow this down more based on the function definition here: https://github.com/tomkp/react-split-pane/blob/master/src/Resizer.js#L27